frontend/src/utils/analytics.ts

```
@@ -11,6 +11,7 @@ interface BaseAnalyticsProperties {
   correlation_id?: string;
   user_id?: string;
   session_id?: string;
+  [key: string]: any;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)